actions/admin_actions.jsx

```diff
@@ -59,6 +59,15 @@ export async function testEmail(config, success, error) {
     }
 }
 
+export async function testS3Connection(config, success, error) {
```

Comment:
Instead of creating a wrapper action, can you just pass the mattermost-redux action in using a connector (index.js for s3_connection_test.jsx)? For example, like this:
* https://github.com/mattermost/mattermost-webapp/blob/master/components/add_users_to_team/index.js#L19
* https://github.com/mattermost/mattermost-webapp/blob/master/components/add_users_to_team/add_users_to_team.jsx#L28

Reply:
done @jwilander, please let me know if this is correct